README.md

```diff
@@ -126,9 +110,9 @@ The following repositories are used by both the current and first generation Kat
 
 | Component | Description | Current | First generation | Notes |
 |-|-|-|-|-|
-| CI | Continuous Integration configuration files and scripts. | [Kata 2.x](https://github.com/kata-containers/ci/tree/main) | [Kata 1.x](https://github.com/kata-containers/ci/tree/master) | |
+| CI | Continuous Integration configuration files and scripts. | [Kata 2.x](https://github.com/kata-containers/ci/tree/master) | [Kata 1.x](https://github.com/kata-containers/ci/tree/master) | |
```

Comment:
Hmm. This is not consistent.

We should move to using `main` on all our repos, for the sake of consistency.
Also, we should not keep references of 1.x, as this is gone for good at this point.

Reply:
This is kinda out of scope of this PR, but I tend to agree, so I've added an extra commit to rip out the 1.x stuff ;)